projects/robots/softbank/nao/worlds/.nao_room.wbproj

```diff
@@ -0,0 +1,11 @@
+Webots Project File version R2021b
+perspectives: 000000ff00000000fd00000003000000000000000000000000fc0100000001fb0000001a0044006f00630075006d0065006e0074006100740069006f006e0000000000ffffffff0000005400ffffff00000001000002fa00000332fc0200000001fb0000001400540065007800740045006400690074006f00720100000015000003320000009d00ffffff0000000300000781000000d8fc0100000001fb0000001a0043006f006e0073006f006c00650041006c006c0041006c006c0100000000000007810000005400ffffff000004810000033200000001000000020000000100000008fc00000000
+simulationViewPerspectives: 000000ff000000010000000200000118000005570100000006010000000100
+sceneTreePerspectives: 000000ff000000010000000200000240000000fc0100000006010000000200
+maximizedDockId: -1
+centralWidgetVisible: 1
+orthographicViewHeight: 1
+textFiles: 0 "controllers/nao_demo_python/nao_demo_python.py"
+consoles: Console:All:All
+renderingDevicePerspectives: NAO:CameraBottom;0;1;0;0
```

Comment:
It would be good to change this overlay position so that it doesn't hide the CameraTop overlay.